pages/linux/fdisk.md

```diff
@@ -0,0 +1,19 @@
+# fdisk
+
+> Manipulate disk partition table.
+
+- List  the  partition  tables for the sda disk and then exit:
```

Comment:
You've got some double spaces in here.

Also "and then exit"/ Does that mean that the other forms of this command don't exit?